templates/generic/link-type/tests/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{#if to_referenceable}}{{kebab_case from_referenceable.name}}-to-{{kebab_case (plural to_referenceable.name)}}.test.ts{{¡if}}.hbs

```diff
@@ -43,21 +48,34 @@ test('link a {{pascal_case from_referenceable.name}} to a {{pascal_case to_refer
 {{#if (eq from_referenceable.hash_type "AgentPubKey")}}
     const baseAddress = alice.agentPubKey;
 {{else}}
+  {{#if (ne from_referenceable.hash_type "AnyLinkableHash")}}
     const baseRecord = await create{{pascal_case from_referenceable.name}}(alice.cells[0]);
+  {{/if}}
   {{#if (eq from_referenceable.hash_type "EntryHash")}}
     const baseAddress = (baseRecord.signed_action.hashed.content as NewEntryAction).entry_hash;
   {{else}}
+    {{#if (eq from_referenceable.hash_type "AnyLinkableHash")}}
```

Comment:
There are still references to `AnyLinkableHash` here that I think need to be replaced with `ExternalHash` now if I'm not mistaken

Reply:
Good catch, I switched to neovim recently, looks like nvim spectre search and replace failed here